CHANGELOG/CHANGELOG-0.9.md

```diff
@@ -0,0 +1,32 @@
+# Release notes for v0.9.0
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since v0.8.0
```

Comment:
There was a v0.8.1. Shouldn't we compare against that?

Reply:
Looking at v0.8 changelog, it's against v0.7.0, not 0.7.1. Btw, branching is odd in this repo, there is no release-0.8 branch

Reply:
..and v0.7.0 is against v0.6.1, not v0.6.0. Updated the PR to show changelog since v0.8.1

Reply:
> ..and v0.7.0 is against v0.6.1, not v0.6.0.

Depends a bit on what came first. It looks like v0.6.1 was created after v0.7.0.

In this case, master is derived from v0.8.1, so keeping the changelog shorter is less confusing.